ddtrace/span.py

```diff
@@ -166,9 +172,11 @@ def duration(self):
 
     @duration.setter
     def duration(self, value):
-        self.duration_ns = value * 1e9
+        # type: (int) -> None
+        self.duration_ns = value * 1e9  # type: ignore
```

Comment:
`* 1e9` turns `duration_ns` into a float; perhaps it's best to either cast `1e9` to `int` or spell out those zeroes (the former would be more readable)?

Reply:
Thanks Gab, I learned something new today! 😮  I'll cast the `duration_ns` as an int in a separate PR but leave the type hinting as is here for now.

Reply:
This is actually an error!!

```
datadog-agent_1  | 2021-03-16 18:44:07 UTC | TRACE | ERROR | (pkg/trace/api/api.go:379 in handleTraces) | Cannot decode v0.4 traces payload: msgp: attempted to decode type "float64" with method for "int"
```